iree/compiler/Conversion/LinalgToSPIRV/KernelDispatchUtils.cpp

```diff
@@ -352,46 +358,130 @@ static LogicalResult getMaliSpecificConfig(linalg::ConvOp op,
   if (!inputType.hasStaticShape() || !outputType.hasStaticShape())
     return failure();
 
-  const int tileWidth = 8;
-  const int tileChannel = 32;
-
-  auto outputShape = outputType.getShape();
   bool isInputTilable = inputType.getDimSize(3) % 4 == 0;
-  bool isOutputTilable = outputShape[0] == 1 &&
-                         outputShape[2] % tileWidth == 0 &&
-                         outputShape[3] % tileChannel == 0;
-  if (!isInputTilable || !isOutputTilable) return failure();
+  if (!isInputTilable) return failure();
 
-  config.workgroupSize = {8, 2, 1};
-
-  SmallVector<int64_t, 4> workgroupLevel = {/*batch=*/0, /*output_height=*/1,
-                                            /*output_width=*/tileWidth,
-                                            /*output_channel=*/tileChannel};
-  tileSizes.emplace_back(std::move(workgroupLevel));
+  static const TileWorkgroupSizePair tileWorkgroupSizePairs[] = {
+      {{1, 8, 32}, {8, 2, 1}},
+  };
 
-  // No tiling at the subgroup level given that we don't use subgroup
-  // level syncrhonization  or shared memory.
-  tileSizes.emplace_back();
+  for (const auto &pair : tileWorkgroupSizePairs) {
+    const auto &tileSize = pair.tileSize;
```

Comment:
Can you spell out the type here and at all other places in change? (Maybe I'm confused between IREE and MLIR style but I thought we wanted to spell out unless the type is obvious)